arangod/IResearch/IResearchRocksDBRecoveryHelper.cpp

```diff
@@ -154,6 +153,40 @@ void IResearchRocksDBRecoveryHelper::PutCF(uint32_t column_family_id,
 
   transaction::StandaloneContext ctx(coll->vocbase());
 
+  // FIXME: check ticks range and possibly omit this step
+  containers::FlatHashSet<IndexId> skipInsert;
```

Comment:
Better to make skipInsert a field, and call clear here.

Reply:
Good point. Done